docs/prerequisite.rst

```diff
@@ -70,10 +70,15 @@ We run the following from the ``src/`` directory
 .. code-block:: console
 
   $ ../bin/mrbob bobtemplates.plone:addon -O example.conference
+ 
+and specify your target version of Plone and python. This will create a directory named ``example.conference`` inside ./src with the basic structure of a generic addon.
```

Comment:
```suggestion
and specify your target version of Plone and python. 
This will create a directory named ``example.conference`` inside ./src with the basic structure of a generic addon.
```

Reply:
One sentence per line